src/fmu/tools/qcforward/_blockedwells_vs_gridprops.py

```diff
@@ -0,0 +1,328 @@
+"""Private module in qcforward, used to compare blocked wells with grid props.
+
+The idea here is to compare blocked wells (cells) with actual values in order
+warn user and/or stop runs if discrepancies are too large.
+
+The resulting report will look like this:
+
+::
+
+             WELL COMPARE(BW:MOD) WARNRULE   STOPRULE  MATCH%  STATUS  NAMETAG
+    INDEX
+        0    A_6  PHIT:PHIT       any<90%    any<70%    99%     OK      MYDATA
+        1    A_6  Facies:FACIES   any<90%    any<70%    91%     OK      MYDATA
+        2    A_5  PHIT:PHIT       any<90%    any<70%    93%     OK      MYDATA
+        3    A_5  Facies:FACIES   any<90%    any<70%    77%     WARN    MYDATA
+        4    A_4  PHIT:PHIT       any<90%    any<70%    88%     WARN    MYDATA
+        5    A_4  Facies:FACIES   any<90%    any<70%    73%     WARN    MYDATA
+        6    all  PHIT:PHIT       all<95%    all<80%    88%     OK      MYDATA
+        7    all  Facies:FACIES   all<95%    all<80%    88%     OK      MYDATA
+
+The input spesification is on the following form if outside RMS:
+
+::
+
+    DATA1 = {
+        "nametag": "MYDATA1",
+        "verbosity": "debug",
+        "path": PATH,
+        "bwells": WELLFILES,
+        "grid": GRIDFILE,
+        "gridprops": [["PHIT", ROFFFILE], ["FACIES", ROFFFILE]],
+        "compare": {"Facies": "FACIES", "PHIT": "PHIT"},  # bwname: modelname
+        "actions": [
+            {"warn": "anywell < 80%", "stop": "anywell < 70%"},
+            {"warn": "allwells < 90%", "stop": "allwells < 80%"},
```

Comment:
`anywell` and `allwells` is this an old syntax? I only see `any` and `all` in the documentation.

Reply:
The code will look at substrings "all" or "any", meaning that both "all", "allwells", "allwhatever" will work equally. I was thinking that "allwells" in this case is more readable than "all"?